GNUmakefile

```diff
@@ -115,9 +115,14 @@ link-git-hooks: ## Install git hooks
 update-atlas-sdk: ## Update the atlas-sdk dependency
 	./scripts/update-sdk.sh
 
-# details on usage can be found in CONTRIBUTING.md under "Creating New Resource and Data Sources"
+# details on usage can be found in CONTRIBUTING.md under "Scaffolding initial Code and File Structure"
```

Comment:
Are there any reasons why we are not adding how to use the script here instead of asking the user to open another file? I'd expect to have here a description on how to use the script and then add in CONTRIBUTING how to use the makefile command